### ISSUE TITLE
Missing hot exit for custom editors?

### ISSUE THREAD

**Issue by roblourens** [microsoft/vscode#86147]
Testing https://github.com/microsoft/vscode/issues/85991

- Disable autosave
- Open a .abc file and a .cat file, modify both, they get dirty indicators.
- Reload the window
- Seems like the .abc file loses the content, and the .cat file was saved?

**Comment by bpasero** [comment]
Requires: https://github.com/microsoft/vscode/issues/88719

**Comment by mjbvz** [comment]
Closing in favor of  #88719
